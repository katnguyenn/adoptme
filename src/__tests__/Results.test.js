import { expect, test } from "@jest/globals";
import { create } from "react-test-renderer";
import Results from "../Components/Results";

test("snapshot with no pets", () => {
    const tree = create(<Results pets={[]} />).toJSON();
    expect(tree).toMatchSnapshot();
})