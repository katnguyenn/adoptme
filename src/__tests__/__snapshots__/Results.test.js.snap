// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`snapshot with no pets 1`] = `
<div
  className="grid gap-4 grid-cols-1 sm:grid-cols-2 lg:grid-cols-3"
>
  <h2>
    No Pets Found
  </h2>
</div>
`;
